templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,21 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
+
+It's easy to send Zulip messages from GitHub Actions!
+
+This is useful:
+
+  * If you want to report the success or failure of an action to Zulip.
 
-  * If you want to report the success or failure of an Action to Zulip.
   * If you want to send output from your actions to Zulip.
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
```

Comment:
I tabbed the instructions as opposed to the use case part, as suggested.
But I'm wondering if we really need to mention the purpose of this integration.
And whether we need to necessarily use tabs for instructions when we've got just a single line.